src/pages/validators.tsx

```diff
@@ -0,0 +1,21 @@
+import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { Validators } from 'src/features/validators/components/Validators';
+import { Mode } from 'src/types';
+import * as ga from '../utils/ga';
+
+const ValidatorsPage: NextPage = () => {
+  const router = useRouter();
+  const onModeChange = (mode: Mode) => {
+    void router.push({
+      pathname: `/${mode}`,
+    });
+    const handleRouteChange = (url: URL) => {
+      ga.pageview(url);
```

Comment:
if we are using regular pages do we still need to record the page view or doesnt ga do that automatically?

Reply:
hmm should be automagical now